src/themes/material/templates/edit-cell-array.tpl.html

```diff
@@ -2,9 +2,7 @@
 	<div class="q-grid-editor q-grid-array">
 		<h2 class="md-title">Edit {{$cell.column.title}}</h2>
 		<div class="q-grid-array-view">
-			<md-chip-list>
-				<md-chip *ngFor="let item of $cell.value" selected="true">{{item}}</md-chip>
-			</md-chip-list>
+			<q-grid-chip-list [values]="$view.edit.cell.value"></q-grid-chip-list>
```

Comment:
just curious, if ngModel is common directive for such cases or not? (instead values)

Reply:
Agree, this is a good idea to use ngModel in this case. Yet, ngModel support requires additional implementation work. I am going to implement this after I've done with other remarks in this review